src/extension.ts

```diff
@@ -1180,17 +1181,25 @@ function getPorts() {
 async function describeKubernetes(explorerNode?: explorer.ResourceNode) {
     if (explorerNode) {
         const nsarg = explorerNode.namespace ? `--namespace ${explorerNode.namespace}` : '';
-        const result = await kubectl.invokeAsync(`describe ${explorerNode.resourceId} ${nsarg}`);
+        const cmd = `describe ${explorerNode.resourceId} ${nsarg}`;
+        const result = await kubectl.invokeAsync(cmd);
+        const refresh = (): Promise<ShellResult | undefined> => {
+            return kubectl.invokeAsync(cmd);
```

Comment:
You can probably simplify `=> { return kubectl.invokeAsync(…); }` to `=> kubectl.invokeAsync(…)` (i.e. lose the braces and the `return`).

Reply:
done.